memberships/static/memberships/js/main.js

```diff
@@ -0,0 +1,33 @@
+const hello = () => console.log("hello");
+var header;
+
+// if scrolled down, make header sticky
+function checkSticky() {
+  if (!header)
+    header = document.getElementsByClassName("header-container")[0];
+  
+  if (window.pageYOffset > header.offsetTop) {
+    header.classList.add("sticky");
+  } else {
+    header.classList.remove("sticky");
+  }
+}
+
+// check user's light/dark mode preference
+function checkDarkMode() {
```

Comment:
I'm a big fan of there always being a toggle for dark/light mode, so we should 100% keep this.

That being said, is it also worth supporting using the users default? I believe this can be done via a media query in CSS https://css-tricks.com/dark-modes-with-css/

Reply:
Agreed - now implemented.